public/externalLibs/sound/soundToneMatrix.js

```diff
@@ -357,11 +357,15 @@ function clear_all_timeout() {
 }
 
 // functions from mission 14
+
+const midi_zero_frequency = 8.17579891564371; // base midi frequency
```

Comment:
Sorry for getting back to this. This number is very odd. The proper definition is A1 which is 440Hz. 
See http://www.cpan.org/modules/by-module/Music/Music-Note-Frequency-0.05.readme

Reply:
Fixed